src/components/spaces/breeding/nest-pair.tsx

```diff
@@ -0,0 +1,196 @@
+    const nestInspectClass = `nest-inspect ${positionClass} ` + (showPairHighlight ? "show" : "");
+    const rightMouseOutlineClass = `mouse-image outline flip ` + (rightMouseCollected ? "show" : "");
+    const leftMouseOutlineClass = `mouse-image outline ` + (leftMouseCollected ? "show" : "");
+
+    return(
+      <div className={nestClass} onClick={this.handleClickNest}>
+        <img
+          src={nestImage}
+          className={`nest-pair-background ${positionClass}`}
+          data-test="nest-pair-background-image"
+        />
+        <img
+          src={nestHoverImage}
+          className={nestBackgroundHoverClass}
+          data-test="nest-pair-background-image"
+        />
+        <div className={nestInspectClass} />
+        <div className={`mouse left ${positionClass}`} onClick={this.handleClickLeftMouse}>
+          <img src={selectionImage} className={leftSelectionClass} data-test="mouse-selection-image" />
+          <img src={leftMouseImage} className={`mouse-image`} data-test="mouse-image" />
+          <img src={mouseOutlineImage} className={leftMouseOutlineClass} data-test="mouse-outline-image" />
+          <img src={heteroImage} className={leftHeteroClass} data-test="mouse-hetero-image" />
+          <img src={leftSexImage} className={sexClass} data-test="mouse-sex-image" />
+        </div>
+        <div className={`mouse right ${positionClass}`} onClick={this.handleClickRightMouse}>
+          <img src={selectionImage} className={rightSelectionClass} data-test="mouse-selection-image" />
+          <img src={rightMouseImage} className={`mouse-image flip`} data-test="mouse-image" />
+          <img src={mouseOutlineImage} className={rightMouseOutlineClass} data-test="mouse-outline-image" />
+          <img src={heteroImage} className={rightHeteroClass} data-test="mouse-hetero-image" />
+          <img src={rightSexImage} className={sexClass} data-test="mouse-sex-image" />
+        </div>
+        <div className={pairLabelClass}>{this.props.nestPair.label}</div>
+      </div>
+    );
+  }
+
+  private isMouseCollected = (id: string | undefined) => {
+    if (id) {
+      const { backpack } = this.stores;
+      const mouse = backpack.collectedMice.find(bpMouse => bpMouse.id === id);
+      if (mouse) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  private getPositionClass = () => {
+    switch (this.props.positionIndex) {
+      case 2:
+        return "left-top";
+      case 4:
+        return "left-middle";
+      case 5:
+        return "left-bottom";
+      case 1:
+        return "right-top";
+      case 3:
+        return "right-middle";
+      case 6:
+        return "right-bottom";
+      default:
+        return "";
+    }
+  }
+
+  private getNestImage = () => {
+    switch (this.props.positionIndex) {
+      case 2:
+        return "../../../assets/curriculum/mouse/breeding/nesting/left-top.png";
```

Comment:
I think if you replace all `../../../assets/` in .ts/.tsx files with `assets/` then it should correctly find the relative path, and will work in branches. See other .tsx files that reference files in `assets/`.

Currently the deployed version at http://connected-bio-spaces.concord.org/branch/nesting-pairs/ doesn't show any of the new assets.

Reply:
Ah thanks!  I was just looking at the deployed version and noticed this.

Reply:
Fixed.